cypress/integration/tests/administrator/repository/git.test.ts

```diff
@@ -74,6 +74,7 @@ describe("Test secure and insecure git repository analysis", { tags: "@tier1" },
 
     after("Perform test data clean up", () => {
         if (hasToBeSkipped("@tier1")) return;
+        login();
```

Comment:
Why do we need to login in after () ?

Reply:
Some states are not cleared after the last test, so when this hooks tries to delete the applications created during the tests, it won't work because the "select all" checkbox is in an undetermined state (not checked nor unchecked).